hcsshim.go

```diff
@@ -14,6 +14,7 @@ import (
 //go:generate go run mksyscall_windows.go -output zhcsshim.go hcsshim.go
 
 //sys coTaskMemFree(buffer unsafe.Pointer) = ole32.CoTaskMemFree
+//sys SetCurrentThreadCompartmentId(compartmentId uint32) (hr error) = iphlpapi.SetCurrentThreadCompartmentId
```

Comment:
I have debated about this to myself. I would like to keep it here as it is simpler than adding this thunk to libnetwork but if everyone feels that we should not do this I can try moving this to libnetwork. It will just make things a little more complicated.